### ISSUE TITLE
alpine/socat container pinned at old version lacking arm64 platform

### ISSUE THREAD

**Issue by dboreham** [testcontainers/testcontainers-java#5279]
As far as I can see the `alpine/socat` image used is defined [here ](https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/containers/SocatContainer.java#L18)as version `1.7.3.4-r0`
That version is quite old and is only available for the amd64 platform on Docker Hub.
The current version `1.7.4.3-r0` is available for other platforms including the increasingly popular arm64.

Used as-is on an arm64 Linux system, it'll fail like this:

```
    2022-04-20 21:01:21.044 [pool-1-thread-1 @coroutine#7] ERROR 🐳 [alpine/socat:1.7.3.4-r0] traceId:  spanId:  - Log output from the failed container:
    standard_init_linux.go:228: exec user process caused: exec format error
```

**Comment by dboreham** [comment]
Note a workaround is to override the default image for `alpine/socat` like so:

```
$ grep socat ~/.testcontainers.properties
socat.container.image=alpine/socat:latest
```

**Comment by kiview** [comment]
Fixed by #5280
